pyproject.toml

```diff
@@ -25,6 +25,8 @@ pytest = "^7.0"
 
 # need lalsuite to compute gmst in injection testing
 lalsuite = "^7.0"
+# useful for creating jupyter notebooks
+ipykernel = "^6.22.0"
```

Comment:
Probably don't want to make this a general requirement of the lib. Possibly as a dev requirement?

Reply:
Ah ok missed that, that makes sense then. The one thing I might suggest is to make this `jupyter` instead since that will install `ipykernel` by default, and then folks who are on LDG can use that to install a kernel and folks who are developing locally can run their own local server